tests/assembly/lto-custom-builtins.rs

```diff
@@ -0,0 +1,45 @@
+// assembly-output: emit-asm
+// compile-flags: --crate-type cdylib -C lto=fat -C prefer-dynamic=no
+// only-x86_64-unknown-linux-gnu
+
+#![feature(lang_items, linkage)]
+#![no_std]
+#![no_main]
+
+#![crate_type = "bin"]
+
+// We want to use customized __subdf3.
+// CHECK: .globl __subdf3
+// CHECK-NEXT: __subdf3:
+// CHECK-NEXT: movq $2, %rax
+core::arch::global_asm!(".global __subdf3", "__subdf3:", "mov rax, 2");
+
+// We want to use __addsf3 of compiler-builtins.
```

Comment:
Aren't this comment, and the one on `__adddf3`, backwards?
It looks like for `__addsf3` the custom function is used, but for `__adddf3` the one from compiler-builtins takes precedence, because the custom one is marked `weak`.

Reply:
It was my mistake.